app/services/statements/query.rb

```diff
@@ -1,21 +1,39 @@
 module Statements
   class Query
-    def initialize(lead_provider:)
-      @lead_provider = lead_provider
+    def initialize
+      @scope = Statement.all
+    end
+
+    def belonging_to(lead_provider:)
+      @scope = @scope
```

Comment:
ah something I missed in the previous pr, could we add the following to the scope `where(output_fee: true)` as that's required for the statements returned in the API? maybe as a different method?

Reply:
also could we maintain the order also missed in the previous pr `scope.order(payment_date: :asc)`?